hathor/p2p/netfilter/targets.py

```diff
@@ -25,6 +25,18 @@ class NetfilterTarget:
     def execute(self, rule: 'NetfilterRule', context: 'NetfilterContext') -> None:
         pass
 
+    def __eq__(self, other: object) -> bool:
+        if not isinstance(other, NetfilterTarget):
+            return NotImplemented
+        eq_type = type(self) == type(other)
+        eq_attrs = True
+        for key in self.__dict__.keys():
```

Comment:
Targets are very simply objects. I feel we should remove this generic comparison algorithm and just implement the `__eq__` method in NetfilterLog. All other targets don't need any customization.

Reply:
This code does not exist anymore